tests/testthat/test-5makePhyloseqFromTreeSummarizedExperiment.R

```diff
@@ -0,0 +1,47 @@
+context("makePhyloseqFromTreeSummarizedExperiment")
```

Comment:
This is an IO function in the most general sense. Please move the testing to the IO test file `test-4IO.R`

Reply:
Moved